astro/.env.local

```
@@ -5,4 +5,6 @@ AUTH_DB_PATH=../db/data/auth.db
 DATABASE_URL=postgresql://mse_radar:mse_radar@localhost:5432/mse_radar
 
 BETTER_AUTH_SECRET=A5NxyTYV7Uhg4AyEHukdvnsMEQ92ObVV
-BETTER_AUTH_URL=http://localhost:4321
\ No newline at end of file
+BETTER_AUTH_URL=http://localhost:4321
+
+RESPONSE_ENCRYPTION_KEY=R8V5Qa7rjLdORwCvPn0AwZnlmL/+O9hWi8uGYtb63uM=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)